### PR TITLE
Download Button Addition in privacy policy

### LINKED ISSUE #996: ✨[Feature Request]:  Adding Private Policy Download button
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

https://www.apple.com/in/legal/privacy/en-ww/

### Use Case

Adding Private Policy Download button

### Benefits

Adding Private Policy Download button

### Add ScreenShots

image

After email it will added.

### Priority

High

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I have starred the repository

### PR DESCRIPTION
**Pull request by Damini2004**
## Description

Download Button Addition in privacy policy

## Related Issues

[Cite any related issue(s) this pull request addresses. If none, simply state “None”]
- Closes #996 

## Type of PR

- [ ] Bug fix
- [x] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): _______________

## Screenshots / videos (if applicable)
[Attach any relevant screenshots or videos demonstrating the changes]

## Checklist

- [x] I have gone through the [contributing guide](https://github.com/Anishkagupta04/RAPIDOC-HEALTHCARE-WEBSITE-/)
- [x] I have updated my branch and synced it with project `main` branch before making this PR
- [x] I have performed a self-review of my code
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x] I have commented my code, particularly in hard-to-understand areas.

## Additional context:
[Include any additional information or context that might be helpful for reviewers.]

### CODE REVIEW

File: Html-Files/privacy_policy_page.html [hunk 1]
```diff
@@ -1100,32 +1121,8 @@ <h4>RAPIDOC Newsletter</h4>
...
+ 
 
-      if (backToTopBtn) {
-        backToTopBtn.addEventListener("click", function () {
-          document.body.scrollTop = 0; // For Safari
-          document.documentElement.scrollTop = 0; // For Chrome, Firefox, edge
-        });
-      }
-
-      // JavaScript to show scroll button when scrolling down
```
Comment (varshith257) [inline Html-Files/privacy_policy_page.html:1115]: Why the scroll bar code deleted?

File: Html-Files/privacy_policy_page.html [hunk 2]
```diff
@@ -1100,32 +1121,8 @@ <h4>RAPIDOC Newsletter</h4>
           }
         }
       }
-    </script>
-    <script>
-      //// JavaScript working of scroll button when scrolling down
```
Comment (varshith257) [inline Html-Files/privacy_policy_page.html:1105]: Same here

**Comment by Damini2004** [comment]
ohkk i will edit it

**Comment by Damini2004** [comment]
@varshith257 changes resolved or not ?.

**Comment by varshith257** [comment]
I didn't get it?

**Comment by Damini2004** [comment]
@varshith257 any conflicts are there in the branch or any changes need to resolve ?

**Pull request merged**